sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/splittabledofn/GrowableOffsetRangeTracker.java

```diff
@@ -140,4 +140,9 @@ public Progress getProgress() {
         totalWork.subtract(workRemaining, MathContext.DECIMAL128).doubleValue(),
         workRemaining.doubleValue());
   }
+
+  @Override
+  public boolean isBounded() {
+    return false;
```

Comment:
Shouldn't this be true if `range.getTo() != Long.MAX_VALUE`?

Reply:
Yeah you are right.